addons/calendar/tests/test_attendees.py

```diff
@@ -16,7 +16,7 @@ def setUpClass(cls):
             'start': datetime(2019, 10, 25, 8, 0),
             'stop': datetime(2019, 10, 27, 18, 0),
         }).with_context(mail_notrack=True)
-        cls.user = new_test_user(cls.env, 'xav', email='em@il.com', notification_type='inbox')
+        cls.user = new_test_user(cls.env, 'xav', email='em@il.com', notification_type='inbox', groups='calendar.group_calendar_secretary')
```

Comment:
don't call this group 'group_calendar_secretary'. It was an example but you don't want to name groups according to the diplom of people. You name the group according to their role in Odoo. For exemple: calendar.group_calendar_event_admin.